docs/index.md

```diff
@@ -245,7 +292,7 @@ Follow these steps:
 	config/Y3tAnotherOrgId_aaa777777_bbbb_4ad3_ccc77777d
```

Comment:
In the latest version, the list will only display the orgIds, there won't be 'config/`f